hadoop-tools/hadoop-azure/src/site/markdown/abfs.md

```diff
@@ -643,6 +643,10 @@ Consult the javadocs for `org.apache.hadoop.fs.azurebfs.constants.ConfigurationK
 `org.apache.hadoop.fs.azurebfs.AbfsConfiguration` for the full list
 of configuration options and their default values.
 
+### <a name="appendblobkeyconfigoptions"></a> Append Blob Directories Options
+### <a name="appendblobkeyconfigoptions"></a> Config `fs.azure.appendblob.key` provides
```

Comment:
should be nested (i.e. ####); duplicate name will confuse link generation. Just cut the "a name" tag from the second line